src/lib.rs

```diff
@@ -732,12 +745,39 @@ impl Config {
         self.cc_algorithm = algo;
     }
 
+    /// Enables support for receiving DATAGRAM frames. When enabled, the
+    /// `max_datagram_frame_size` transport parameter is set to 65536 as
+    /// recommended by the current draft of the standard.
+    ///
+    /// The default is `false`.
+    pub fn set_dgram_frames_supported(&mut self, supported: bool) {
```

Comment:
Fwiw, I really like this suggestion.

Reply:
There's a slight problem with this suggestion because what does `enable_dgram()` mean? Right now we allow an endpoint to opt into a transport feature, and configure some internal queue lengths. 

So what about an application that wants to only send DATAGRAMS and control it's send queue size? It would be counter intuitive to call `enable_dgram(usize, usize)`. I suggest `enable_dgram(bool, usize, usize)`